controller/array_action/messages.py

```diff
@@ -62,3 +62,7 @@
 ObjectIsStillInUseError_message = "Object {0} is still in use by {1}"
 
 InvalidCliResponseError_message = "Invalid CLI response. Details : {0}"
+
+NotEnoughSpaceInPoolError_message = "Not enough space in pool"
```

Comment:
I dont have it..

Reply:
what do you mean? why can't a mediator fill the pool value?

Reply:
sure, but only to pass the pool parameter to the mediators for log purposes seems too much

Reply:
why would the pool parameter be passed to the mediators? the controller server doesn't have the pool parameter

Reply:
controller gets it in the parameters var in the request.
mediators dont have it and they are not getting the volume during the expansion so they cant query it.

Reply:
then could you please link + quote the spec where it says the controller gets a "parameters" var in the request?
which mediators are not getting the volume during expansion and why can't they query it?

Reply:
I looked again and the controller dont get it.
ds8k dont get it.
why should it get it if it dosent need it?

Reply:
ok, I will put it in

Reply:
`it` might be confusing.
ds8k mediator should get the pool value because the controller server needs it, that's the purpose of return values.
so the ds8k mediator will have to get the pool value somehow eventually because that's what the `ControllerExpandVolume` flow requires (no matter how we arrange the code).

Reply:
done